packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -88,7 +88,7 @@ function toPodInfo(pod: V1Pod): PodInfo {
     Namespace: pod.metadata?.namespace || '',
     Networks: [],
     Status: pod.status?.phase || '',
-    engineId: 'kubernetes',
+    engineId: contextName || 'kubernetes',
```

Comment:
```suggestion
    engineId: contextName ?? 'kubernetes',
```

just a suggestion, I saw that we should use nullish coalescing runtime operatorbut you can keep || a linter rule will come soon